nvme.c

```diff
@@ -4450,6 +4450,19 @@ static int get_feature_id(struct nvme_dev *dev, struct feat_cfg *cfg,
 	return nvme_cli_get_features(dev, &args);
 }
 
+static bool nvme_status_invalid_field(int status)
+{
+	enum nvme_status_type type = NVME_STATUS_TYPE_NVME;
+	unsigned int value = NVME_SC_INVALID_FIELD;
```

Comment:
Just use the values directly in `nvme_status_equals()`, no need to define them as local variable.

Reply:
Noted and will do fix. (I just thought to shorten the 'nvme_status_equals()` line since the definition value names NVME_S*** length not short.)